.github/workflows/ci_codebuild.yml

```
@@ -0,0 +1,44 @@
+---
+name: s2nCodeBuild
+
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        openssl_version:
+          - openssl-1.0.2
+          - openssl-1.0.2-fips
+          - openssl-1.1.1
+        gcc_version:
+          - 6
+          - 9
+      fail-fast: true
+    steps:
+      - uses: actions/setup-node@v1
+      - name: S2n Unit CodeBuild
+        uses: aws-actions/aws-codebuild-run-build@v1
```

```suggestion
        uses: aws-actions/aws-codebuild-run-build@master
```

We haven't published a release tag yet. For this initial testing just pull from `master`.